app/src/main/java/de/tum/in/tumcampusapp/component/other/generic/drawer/DrawerMenuHelper.kt

```diff
@@ -86,7 +85,6 @@ class DrawerMenuHelper(private val mContext: Context, private val mDrawerLayout:
                 SideNavigationItem(R.string.menues, R.drawable.ic_cutlery, CafeteriaActivity::class.java, false, false),
                 SideNavigationItem(R.string.study_rooms, R.drawable.ic_group_work, StudyRoomsActivity::class.java, false, false),
                 SideNavigationItem(R.string.roomfinder, R.drawable.ic_place, RoomFinderActivity::class.java, false, false),
-                SideNavigationItem(R.string.mvv, R.drawable.ic_mvv, TransportationActivity::class.java, false, false),
```

Comment:
I think @kordianbruck vetoed the removal, but IMO we want to improve notifications and cards for that.

Reply:
I'd additionally remove that "opening hours" activity, since we really don't update those too often

Reply:
It was @kordianbruck who also suggested the removal while keeping the card and the details activity.